src/components/Capture/index.js

```diff
@@ -16,17 +16,19 @@ const BackDocumentCapture = options => <DocumentCapture {...options} />
 
 BackDocumentCapture.defaultProps = { side: 'back' }
 
-const FaceCapture = options =>
-  <Capture autoCapture={false} {...options} />
+const FaceCapture = appendToTracking(options =>
+  <Capture autoCapture={false} {...options} />, 'capture')
 
 FaceCapture.defaultProps = {
   useWebcam: true,
   method: 'face',
   side: null
 }
 
+FaceCapture.isFullScreen = true
```

Comment:
is this still necessary then?

Reply:
Yes, it is. Otherwise the full screen options will be applied to the confirm screen of the face step too.

Reply:
if we are using the callback, let's just do that.